bin/ofs/src/bin/ofs.rs

```diff
@@ -18,7 +18,8 @@
 use anyhow::Result;
 use clap::Parser;
 
-#[tokio::main]
+// FIXME: https://github.com/Sherlock-Holo/fuse3/issues/91
```

Comment:
Please don't link to external issues directly. Create an issue in opendal and add this link as references.